backend/config/initializers/devise.rb

```diff
@@ -0,0 +1,290 @@
+# frozen_string_literal: true
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
```

Comment:
i tried to comment this already: i would add `config.secret_key` and `config.pepper` (now commented out) as and `config.mailer_sender ` ENVs to this file. 
Plus i would delete the rest of the comments. or at least the description.